website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+page_title: Filtering Audit
+description: |-
+  How to filter audit entries in Vault.
+---
+
+[filter syntax]: /vault/docs/concepts/filtering
+
+# Filtering audit entries
+
+This page describes how to use filters with Vault audit devices to fine tune which
+audit entries should be written to an audit log.
+
+Starting in Vault 1.16.0, audit devices can be enabled with a `filter` option which
+is used to evaluate audit entries and decide if they should be written to an audit log.
+Please ensure you are familiar with the [filtering concept](/vault/docs/concepts/filtering)
```

Comment:
This might still be forthcoming but I'd also love to get some examples here to help make everything more concrete.